### PR TITLE
Upgrade all projects to .NET 8.0rc2

### PR DESCRIPTION
**Pull request by william-liebenberg**
Update all dependencies

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "7.0.200",
+    "version": "8.0.100-rc.2.23502.2",
     "rollForward": "latestFeature",
     "allowPrerelease": false
```
Comment (danielmackay) [inline global.json:5]: should `allowPrerelease` be set to `true`?

File: src/WebApi/wwwroot/api/specification.json [hunk 2]
```diff
@@ -109,15 +121,15 @@
         "type": "object",
         "additionalProperties": false,
         "properties": {
-          "id": {
+          "Id": {
```
Comment (danielmackay) [inline src/WebApi/wwwroot/api/specification.json:124]: Why are these Pascal case now?  Do we need to add configuration to get it back to camel case?

**Review by danielmackay — CHANGES_REQUESTED** [review]
2 comments

File: global.json (hunk 1 above)
Comment (william-liebenberg) [reply global.json]: yes indeed :) it should be for now.

fixed!

File: src/WebApi/wwwroot/api/specification.json (hunk 2 above)
Comment (william-liebenberg) [reply src/WebApi/wwwroot/api/specification.json:124]: As discussed and explored, the configuring the `"defaultPropertyNameHandling": "CamelCase"` setting in nswag.json is deprecated. 

Controlling the serializer casing now for the openapi/swaggerdoc seems to be quite difficult.

However, the PascalCase 'default' does not seem to be a problem for generating client code (C# / TypeScript).

We will proceed with this new default for now.

**Review by danielmackay — APPROVED** [review]
LGTM

**Pull request merged**